codeHF/config_tasks.sh

```diff
@@ -28,6 +28,8 @@ DATABASE_O2="workflows.yml"
 MAKE_GRAPH=0        # Make topology graph.
 
 # Activation of O2 workflows
+# Event selection
+DOO2_EVSEL=1        # event-selection and timestamp
```

Comment:
Should be `0` by default.

Reply:
Indeed this was a leftover of the tests 😄 fixed!